### PR TITLE
ENG-1461: changing image provider path

### PR DESCRIPTION
**Pull request by gReis89**
changing image provider to retrieve resources from `cmsresources/static/widget-icon` as per @alepintus request.

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **greis89**

If they are not already assigned, you can assign the PR to them by writing `/assign @greis89` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fapp-builder).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/app-builder/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: src/ui/widgets/common/WidgetIcon.js
Comment (gidesan) [inline src/ui/widgets/common/WidgetIcon.js:7]: `getResourcePath` is now not only used for MFE, so it makes sense to move it elsewhere

File: src/ui/widgets/common/WidgetIcon.js
Comment (gidesan) [inline src/ui/widgets/common/WidgetIcon.js:9]: please remove comments

**Review by gidesan — CHANGES_REQUESTED** [review]

File: src/ui/widgets/common/WidgetIcon.js
Comment (gReis89) [reply src/ui/widgets/common/WidgetIcon.js]: moved to a resourcePath file

**Review by gidesan — APPROVED** [review]

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/app-builder/pull/1016#pullrequestreview-598537098" title="Approved">gidesan</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fapp-builder).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando/app-builder/blob/master/OWNERS)~~ [gidesan]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
